tnnt/templates/clanmgmt.html

```diff
@@ -0,0 +1,125 @@
+{% load static %}
+<!doctype html>
+
+<html>
+
+<head>
+  <title>TNNT::Clan Management</title>
+  {% include "headlinks.html" %}
+</head>
+
+<body>
+
+{% include "header.html" with page="clanmgmt" %}
+
+<h2>Clan Management</h2>
+
+<article>
+{% if not user.is_authenticated %}
+  <p> You are not logged in. </p>
+  <p><a href="/login">Log in here.</a> </p>
+{% else %}
+  <p>Logged in as {{ user.username }}. <a href="{% url 'logout' %}">Log Out</a></p>
+
+  <hr>
+  {% if errmsg is not None %}
+    <p class="errmsg">Error: {{errmsg}}</p>
+  {% endif %}
+
+  {% if clan is None %}
+    <p>You are not in a clan right now.</p>
+    <form method="post">
+        {% csrf_token %}
+        {{ create_clan_form }}
+        <button name="create_clan" type="submit">Create</button>
+    </form>
+  {% else %}
+    <h3>{{clan.name}}</h3>
+    <form method="post" class="inline-form">
+      {% csrf_token %}
+        <button name="leave" type="submit" {% if members.count == 1 %}disabled{% endif %}>
```

Comment:
Wouldn't we want to remove the clan if the last player left?

Reply:
Submitting a "leave" POST request when the player is the sole clan member will indeed disband the clan (it just calls the disband_clan function), but in the UI there's a big "Disband Clan" button anyway, so there's no reason to confusingly have "Leave" up there as well.

Reply:
Ahhhh okay, I gotchu.